experiment/audit-owners/main.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This application reads kubernetes OWNERS files within a repo and compares it
+// to collaborators for a repo to find those listed in OWNERS files who are not
+// collaborators
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/ghodss/yaml"
+	"k8s.io/test-infra/prow/github"
+)
+
+var (
+	// The root location of the repo to start walking the filesystem
+	repoRoot string
+
+	// The name of the GitHub org to use when querying for collaborators
+	githubOrg string
+
+	// The name of the GitHub repo to use when querying for collaborators
+	githubRepo string
+)
+
+func init() {
+	flag.StringVar(&repoRoot, "srcdir", "", "The location of the repo on the local filesystem to start inspecting")
+	flag.StringVar(&githubOrg, "org", "", "The name of the GitHub org to use when querying for collaborators")
```

Comment:
Instead of passing a single org/repo combo, you could:
1. pass a url to a prow deployment
2. get and unmarshal its live config
3. get all repos registered for that prow deployment 
4. inspect all owners files in all repos

You can find an example for 1-2 in [refresh](https://github.com/kubernetes/test-infra/tree/master/experiment/refresh), a prow external plugin. 3-4 is logic that you would have to write.

Reply:
> You can find an example for 1-2 in refresh

Sorry this wasn't the example I wanted to show you. The correct one is [fallbackcheck](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/tot/fallbackcheck).